iis-wf/iis-wf-primary/src/test/resources/eu/dnetlib/iis/wf/primary/processing/sampledataproducer/oozie_app/workflow.xml

```diff
@@ -422,6 +426,11 @@
 				eu/dnetlib/iis/wf/primary/processing/sampledataproducer/output/document_to_research_initiatives.json}
 			</arg>
 			<arg>-Idocument_to_research_initiatives=${workingDir}/exported/document_to_research_initiatives</arg>
+            <arg>-C{document_to_community,
+                eu.dnetlib.iis.export.schemas.DocumentToConceptIds,
```

Comment:
Is using `eu.dnetlib.iis.export.schemas.DocumentToConceptIds` here is not an error e.g. shouldn't we use something like `DocumentToCommunityId` here ?

Reply:
Well it was made on purpose because of Avro schema resolution requirement mentioned on:

http://avro.apache.org/docs/current/spec.html#Schema+Resolution

literally:
> both schemas are records with the same (unqualified) name